misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -223,17 +210,6 @@ internal class RealTransacter private constructor(
     }
   }
 
-  /**
-   * Returns a string describing the most recent attempt. This includes whether the attempt used the
-   * same connection which might help in diagnosing stale data problems.
-   */
-  private fun attemptNote(attempt: Int): String {
-    if (attempt == 1) return "attempt 1"
-    val latestSession = threadLatestSession.get()!!
-    if (latestSession.sameConnection) return "attempt $attempt, same connection"
-    return "attempt $attempt, different connection"
-  }
```

Comment:
This function required us to remember the previous session, which I think was a pretty sketchy thing for us to do. Not remembering it makes so many other things simpler!